scripts/getPDWorks.py

```diff
@@ -0,0 +1,46 @@
+import os
+import json
+
+from model import Record
+from managers import DBManager
+
+import json
+
+jsonFile = 'UofM_FullAccess_CSV.json'
+
+fullAccessArray = []
+
+def main():
+
+    '''Retrieving former LA works that are now PD'''
+
+    dbManager = DBManager(
+        user= os.environ.get('POSTGRES_USER', None),
+        pswd= os.environ.get('POSTGRES_PSWD', None),
+        host= os.environ.get('POSTGRES_HOST', None),
+        port= os.environ.get('POSTGRES_PORT', None),
+        db= os.environ.get('POSTGRES_NAME', None)
+    )
+
+    dbManager.generateEngine()
+
+    dbManager.createSession()
```

Comment:
What are your thoughts on using a __enter__, __exit__ methods so we can use a with statement?

Reply:
I'm open to seeing how that would be implemented.

Reply:
sounds good!